source/scripting/Tasks.cs

```diff
@@ -185,6 +186,22 @@ public void GoTo(Vector3 position, bool ignorePaths = false, int timeout = -1)
 			}
 		}
 
+	    public void GoToCoordWhileAiming(Vector3 position, Vector3 aimCoord, bool shooting, FiringPattern pattern)
+	    {
+	        if (shooting)
+	        {
+	            Function.Call(Hash.TASK_GO_TO_COORD_WHILE_AIMING_AT_COORD, _ped.Handle, position.X, position.Y,
+	                position.Z, aimCoord.X, aimCoord.Y, aimCoord.Z, 2f, 1, 0x3F000000, 0x40800000, 1, 0, 0,
+	                (uint) pattern);
```

Comment:
This is a minor one but its a pet hate of mine. that 0x3F000000 should be 0.5f, and 0x40800000 should be 4.0f. writing them as floats makes it more understandable for people when reading the code